_pytest/_code/source.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import generators
+from __future__ import absolute_import, division, generators, print_function
```

Comment:
I agree, but I think that refactoring would be out of scope for this PR, don't you agree?